superset-frontend/src/cccs-viz/plugins/plugin-chart-ag-grid/src/ag-grid/ContextMenu/MenuItems/SubmitToAssemblyLineMenuItem.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Menu } from 'src/components/Menu';
+import Icon from '@ant-design/icons';
+import AlSvg from '../../../cccs-grid/images/al.svg';
+
+interface SubmitToAssemblyLineMenuItemProps {
+  label: string;
+  data: string[];
+  onSelection: () => void;
+  key?: string;
+  disabled?: boolean;
+  isContextMenu?: boolean;
+  contextMenuY?: number;
+}
+
+export default function SubmitToAssemblyLineMenuItem(
+  props: SubmitToAssemblyLineMenuItemProps,
+) {
+  // Handler for clicking on the item
+  const onClick = () => {
+    // Loop through each item in the data array and open a new window for each URL
+    // Using encodeURIComponent to prevents URL injection issues and ensures that special characters in the data strings are correctly interpreted by the URL
+    props.data.forEach(item => {
+      const url = `https://malware-stg.cyber.gc.ca/submit?input=${encodeURIComponent(
```

Comment:
Wow I missed this entirely. +1 for this.

Reply:
Should I just use ASSEMBLY_LINE_URL: malware-pb.cyber.gc.ca and exclude https://malware-stg.cyber.gc.ca/ altogether?

Reply:
I think this value should come from an environment variable. U would have a different url.

Reply:
@nz970 were you asking if you should point pb-stg to AL prod? Because it doesn't look like we use malware-stg in our deployment currently -- we just point to PB or U prod depending where we are. 

We do point superset-stg to alfred-stg, so I lean toward adding malware-stg to our deployment and pointing superset-stg to that. The one caveat is that malware-stg seems to require the AL team to approve access for anyone who tries to connect and I'm not sure how long that permission persists. So it might be a pain point for development? @cccs-joel -- thoughts?

Reply:
What I'm wondering is that currently in the hogwarts deployment repo we have ASSEMBLY_LINE_URL: malware-pb.cyber.gc.ca 
Should I just use this value for the url portion?  Or should I create a whole new env?  Also, does the url used in OpenInAssemblyLine change dynamically?  If so, the value is derived from ASSEMBLY_LINE_URL

Reply:
From what I can see, ASSEMBLY_LINE_URL has pb and unclass versions in deployment repo currently

Reply:
Anyhow I changed to code to use the assembly line url env

Reply:
Ideally, PB-Stg would use AL PB-stg and but yes, Malware PB-Stg seems to require a special permission, I asked for it a week ago and still waiting, so ok to use Malware PB for Superset PB-Stg. The point here to use an environment variable to get the url.

Reply:
Just to make sure this is documented -- AL has no PB-stg instance. :)